### DISCUSSION TITLE
PhotoPrism browser tab name

### DISCUSSION THREAD

**Discussion by joon-im** [tteck/Proxmox#2428] [Help Needed]
Hello there! 

When I install the PhotoPrism script, everything is fine except for when I open new tabs on my web browser, the "name" of the tab is the url for tteck's website instead of something like "PhotoPrism". Other scripts don't have this issue. Here is what it looks like on my computer: 

![Screenshot 2024-02-04 at 1 14 25 PM](https://github.com/tteck/Proxmox/assets/32493276/c99830dc-dde4-431d-b9dc-98bf88ee802b)

Is there something I am doing incorrectly? Any advice would be appreciated.

**Comment by tteck** [comment]
Try Chrome.

**Comment by joon-im** [comment]
Having the same issue with Chrome on Mac: 

![Screenshot 2024-02-04 at 1 28 24 PM](https://github.com/tteck/Proxmox/assets/32493276/a75c34d6-8d4b-43cc-a124-a371377367f0)

**Comment by joon-im** [comment]
I solved this issue by changing the PHOTOPRISM_SITE_CAPTION variable found in the .env file. The .env file was found here: 

/opt/photoprism/config
